entries/d/datto.com.json

```diff
@@ -0,0 +1,18 @@
+{
+  "Datto": {
+    "domain": "datto.com",
+    "tfa": [
+      "totp",
+      "custom-software"
+    ],
+    "custom-software": [
+      "duo"
+    ],
+    "documentation": "https://help.datto.com/s/article/KB360032979352",
+    "keywords": [
+      "remote",
+      "marketing",
```

Comment:
I don't really see how this fits into marketing?

Reply:
I saw a marketing service called [MarketNow](https://www.datto.com/partners/marketnow) on their site but it looks like it's some sort of partnering company and not a main service from Datto.